test/test_go.rb

```diff
@@ -1935,61 +1903,54 @@ def test_dir_completion
     tmux.send_keys 55
     tmux.until { |lines| lines.match_count == 1 }
     tmux.send_keys :Enter
-    tmux.until(true) { |lines| lines[-1] == 'cd /tmp/fzf-test/d55/' }
+    tmux.until { |lines| lines[-1] == '$ cd /tmp/fzf-test/d55/' }
     tmux.send_keys :xx
-    tmux.until { |lines| lines[-1] == 'cd /tmp/fzf-test/d55/xx' }
+    tmux.until { |lines| lines[-1] == '$ cd /tmp/fzf-test/d55/xx' }
 
     # Should not match regular files (bash-only)
     if self.class == TestBash
       tmux.send_keys :Tab
-      tmux.until { |lines| lines[-1] == 'cd /tmp/fzf-test/d55/xx' }
+      tmux.until { |lines| lines[-1] == '$ cd /tmp/fzf-test/d55/xx' }
     end
 
     # Fail back to plusdirs
     tmux.send_keys :BSpace, :BSpace, :BSpace
-    tmux.until { |lines| lines[-1] == 'cd /tmp/fzf-test/d55' }
+    tmux.until { |lines| lines[-1] == '$ cd /tmp/fzf-test/d55' }
     tmux.send_keys :Tab
-    tmux.until { |lines| lines[-1] == 'cd /tmp/fzf-test/d55/' }
+    tmux.until { |lines| lines[-1] == '$ cd /tmp/fzf-test/d55/' }
   ensure
     FileUtils.rm_r '/tmp/fzf-test'
   end
 
   def test_process_completion
-    tmux.send_keys 'sleep 12345 &', :Enter
-    lines = tmux.until { |lines| lines[-1].start_with? '[1]' }
-    pid = lines[-1].split.last
     tmux.prepare
-    tmux.send_keys 'C-L'
+    tmux.send_keys 'sleep 12345 &', :Enter
+    pid = nil
+    tmux.until { |lines| pid = /(?<= )\d+/.match lines[-2] }
     tmux.send_keys 'kill ', :Tab
-    tmux.until { |lines| lines.match_count.positive? }
+    tmux.until { |lines| lines.item_count.positive? }
     tmux.send_keys 'sleep12345'
-    tmux.until { |lines| lines.any_include? 'sleep 12345' }
+    tmux.until { |lines| lines.match_count == 1 }
```

Comment:
How about changing this to `lines.match_count >= 1`? We can't be 100% sure if our process is the only match on the system.

Reply:
If there's more than one match can we be sure we'll the right one?

Reply:
No, but it will avoid false negatives. Any reason you decided to remove `any_include?` which should be more robust in that regard? If we would really like to avoid false positives, we could generate a large random integer instead of `12345` and search for that number.

Reply:
> No, but it will avoid false negatives.

Sorry, I didn't think carefully enough! You're right: I've changed it to `lines.match_count >= 1` :+1:

> Any reason you decided to remove `any_include?` which should be more robust in that regard?

Is it more robust? It checks that the process is among the matches but another match might still be selected?

Reply:
It is not perfect, but I'm sure we can agree that it is better than not checking at all.

Reply:
I see your point, how about `lines[5].end_with?(' sleep 12345')`?